tailwind.config.js

```diff
@@ -6,8 +6,30 @@ module.exports = {
     "./src/components/**/*.{js,ts,jsx,tsx}",
     "./src/**/*.{js,ts,jsx,tsx}",
   ],
+
   theme: {
-    extend: {},
+    extend: {
+      backgroundImage: {
+        hero: "url('/photos/hero-img.svg')",
+      },
+    },
+
+    screens: {
+      sm: "340px",
+      // => @media (min-width: 640px) { ... }
+
+      md: "768px",
+      // => @media (min-width: 768px) { ... }
+
+      lg: "1024px",
+      // => @media (min-width: 1024px) { ... }
+
+      xl: "1280px",
+      // => @media (min-width: 1280px) { ... }
+
+      "2xl": "1536px",
+      // => @media (min-width: 1536px) { ... }
+    },
```

Comment:
[ ] Don't commit commented code to master

Reply:
Deleted commented code .